include/swift/IDE/RefactoringKinds.def

```diff
@@ -50,6 +50,8 @@ CURSOR_REFACTORING(ConvertToDoCatch, "Convert To Do/Catch", convert.do.catch)
 
 CURSOR_REFACTORING(TrailingClosure, "Convert To Trailing Closure", trailingclosure)
 
+CURSOR_REFACTORING(DocCommentBoilerplate, "Generate Documentation Comment", doc.comment.boilerplate)
```

Comment:
To be more precise, we can change "Generate Documentation Comment" to "Generate Documentation Comment Template"